requirements.txt

```diff
@@ -5,8 +5,8 @@ pandas>=1.2.0
 scikit-learn>=0.22
 matplotlib>=3.0.0
 # Install only to use SQLExecutor
-# psycopg2>=2.8.5
-# psycopg2-binary>=2.8.5
+psycopg2>=2.8.5
```

Comment:
Does our code depend on these or are they user-specified? In the latter case, we can make users install these dependencies on their own. Since users have a choice of using `psycopg2` or `sqlalchemy`, we can simply let the user install these connections on their own.

Reply:
Merging this in for now, but we should check that `psycopg2` doesn't break for installing on different systems.